starterkit_app/.vscode/launch.json

```diff
@@ -29,7 +29,7 @@
             },
             "args": [
                 "--dart-define-from-file",
-                ".secrets/${input:appEnvironment}.json",
```

Comment:
Was this change intentional? It seems to contradict your docs.

Reply:
Yes this is intentional. I originally used the actual secrets file when running tests. I changed it and used a test asset file `test/assets/test_secrets.json` instead.